docs/sources/operators-guide/get-started/_index.md

```diff
@@ -98,6 +98,8 @@ store_gateway:
 ```
 <!-- prettier-ignore-end -->
 
+> **Note**: Grafana Mimir includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Mimir cluster to a remote statistics server. If possible, we ask you to keep the usage reporting feature enabled and help us understand more about how the open source community runs Mimir. In case you want to opt-out, refer to [Disable the anonymous usage statistics reporting]({{< relref "../about-anonymous-usage-statistics-reporting.md#disable-the-anonymous-usage-statistics-reporting" >}}).
```

Comment:
```suggestion
> **Note**: Grafana Mimir includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Mimir cluster to a remote statistics server. If possible, we ask you to keep the usage reporting feature enabled to help us understand more about how the open source community runs Mimir. In case you want to opt-out, refer to [Disable the anonymous usage statistics reporting]({{< relref "../about-anonymous-usage-statistics-reporting.md#disable-the-anonymous-usage-statistics-reporting" >}}).
```

Reply:
The sentence has been copied from another page, where it was reviewed by Ursula. I will let them follow up on this.